src/components/RecipeIngredient.vue

```diff
@@ -50,19 +50,32 @@ li {
         list-style-type: none;
         font-variant: small-caps;
     }
-    
+
+    li.unindented {
+        position: relative;
+        left: -1.25em;
```

Comment:
This can cause trouble. After #515 it might be better to do `margin-left: 0px;` instead.